src/modules/user-settings/components/terminal-picker.tsx

```diff
@@ -25,7 +25,7 @@ export const TerminalPicker = ( {
 	);
 
 	return (
-		<SettingsFormField label={ __( 'Shell' ) }>
+		<SettingsFormField label={ __( 'Terminal application' ) }>
 			<SelectControl
 				value={ value }
 				onChange={ ( newValue ) => onChange( newValue as SupportedTerminal ) }
```

Comment:
```suggestion
			<SelectControl< SupportedTerminal >
				value={ value }
				onChange={ ( newValue ) => onChange( newValue ) }
```
Optional nit, but `as` operators always irk me. We could make the corresponding change in `src/modules/user-settings/components/editor-picker.tsx`

Reply:
Can we have another issue or PR for that?

Reply:
Sure. It's a small concern, but I could put together a separate PR to fix this and a few other type-related changes